openapi.yml

```diff
@@ -196,19 +196,6 @@ components:
       required:
         - identifier
         - type
-    Administrative:
-      type: object
-      additionalProperties: false
-      properties:
-        hasAdminPolicy:
-          $ref: '#/components/schemas/Druid'
-        releaseTags:
-          description: Tags for release
-          type: array
-          items:
-            $ref: '#/components/schemas/ReleaseTag'
-      required:
-        - hasAdminPolicy
```

Comment:
The two changes to the OpenAPI spec are:

1. Split catalog link schema into folio and symphony schemas (the meat of this PR); and
2. Alphabetize all the schemas (requested by @justinlittman in past round of code review).

Reply:
I do not recommend trying to review the changes to this file. Suffice to say that the openapi validation in CI succeeds, the model changes generated from the spec are all as expected, and all the specs pass?

Reply:
is this due to alphabetization of the types?  Perhaps in a perfect world that would have been its own PR but I'll allow it.  :-D